app/src/androidTest/java/org/dhis2/usescases/datasets/DataSetTest.kt

```diff
@@ -105,6 +97,19 @@ class DataSetTest : BaseTest() {
         checkPivotOptions()
     }
 
+    @Test
```

Comment:
nice!, Maybe we can update the formConfigurationTest to go throuhg all of the sections isntead of having to but for now it's a great test!